specifications/android.md

```diff
@@ -35,7 +35,7 @@ The connector library MUST expose the messaging broadcast receiver, allowing the
 The library itself does not declare any activity, service or receiver in the manifest. The end user application integrating the library MUST expose a broadcast receiver with the following actions:
 * org.unifiedpush.android.connector.NEW_ENDPOINT
 * org.unifiedpush.android.connector.UNREGISTERED
-* org.unifiedpush.android.connector.MESSAGE
+* org.unifiedpush.android.connector.MESSAGEv2
```

Comment:
You should probably deprecate the other one, right? Or why even that. Why not keep both?